ods_ci/run_interop.sh

```diff
@@ -0,0 +1,48 @@
+!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

Reply:
following this error:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58481/rehearse-58481-periodic-ci-red-hat-data-services-ods-ci-master-rhoai2.16-ocp4.18-lp-interop-rhoai-interop-aws/1867044695671050240#1:build-log.txt%3A96

Reply:
Updated.